### PR TITLE
Cf

### PR DESCRIPTION
**Pull request by yinlinchen**
**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2242) (:star:)

# What does this Pull Request do? (:star:)
Add Cloudformation one-click button.

# What's the changes? (:star:)
Add Cloudformation one-click button with instruction.

# How should this be tested?
1. See the update readme.md
2. Deploy this application 
3. Make sure it's clear and accurate, proof-read, grammar, etc

# Interested parties
Tag (@vt-digital-libraries-platform/diglib-dev ) interested parties

(:star:) Required fields

### CODE REVIEW

**Review by tingtingjh — APPROVED** [review]
I tested using both one-click CloudFormation button and AWS CLI and successfully created/updated stack.

I approve.

**Review by whunter — APPROVED** [review]
I was able to successfully create the stack using the button and deploy it. Pretty cool. It looks like I don't have write access to this repo though :( so I can't merge it.

**Pull request merged**
